tests/compatibility-tests/src/test/java/org/apache/activemq/artemis/tests/compatibility/SerializationTest.java

```diff
@@ -91,15 +91,15 @@ public void afterTest() {
    @Test
    public void testSerializeFactory() throws Throwable {
       File file = serverFolder.newFile("objects.ser");
-      callScript(senderClassloader, "serial/serial.groovy", file.getAbsolutePath(), "write");
-      callScript(receiverClassloader, "serial/serial.groovy", file.getAbsolutePath(), "read");
+      callScript(senderClassloader, "serial/serial.groovy", file.getAbsolutePath(), "write", sender);
+      callScript(receiverClassloader, "serial/serial.groovy", file.getAbsolutePath(), "read", receiver);
    }
 
    @Test
    public void testJBMSerializeFactory() throws Throwable {
       File file = serverFolder.newFile("objectsjbm.ser");
-      callScript(senderClassloader, "serial/jbmserial.groovy", file.getAbsolutePath(), "write");
-      callScript(receiverClassloader, "serial/jbmserial.groovy", file.getAbsolutePath(), "read");
+      callScript(senderClassloader, "serial/jbmserial.groovy", file.getAbsolutePath(), "write", sender);
```

Comment:
what's the sender/receiver for?

Reply:
@jbertram mistake?

Reply:
It's the version used by the script to determine which destination constructor to use.

Reply:
@jbertram ignore me.. lol... I read "sender" instead of sender.. 
I"m getting old I guess